docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
I don't particularly agree with case A because we should not ignore configs. It makes transitioning out of the old way harder. It also makes the config model more restrictive (compared to what we have today) because now I can't have system-scoped properties defined. Is my interpretation correct about not supporting system-scoped configs ?

With Case B, I like the idea of unifying configs for metadata streams under the same umbrella. However, we should still allow overrides at system-scope. Let's say Kafka introduces exactly-once producer and we want to leverage that for all out metadata systems. That implies, I have to set 2 or more properties at stream-scope (streams.<coordinator-stream-name>.producer.<property>, streams.<checkpoint-stream-name>.producer.<property>, instead of one system-scoped property for kafka. Also, metadata stream names are generated by samza during runtime. So letting expecting the user to statically define it complicates our model, doesn't it? 

Please correct me if I have understood your proposed solutions incorrectly :)

I saw you that created JIRAs for follow-up on changes. However, I think we need to be on the same page before executing the jiras :)